tests/cxx/unit_tests/ghostfragment/fragmenting/dimer_maker.cpp

```diff
@@ -0,0 +1,70 @@
+#include "../test_ghostfragment.hpp"
+#include <ghostfragment/property_types/fragmented_nuclei.hpp>
+#include "hydrocarbon/hydrocarbon.h"
+
+using namespace ghostfragment;
+using namespace testing;
+
+using my_pt        = pt::FragmentedNuclei;
+using frags_pt     = pt::FragmentedNuclei;
+using input_type   = chemist::Nuclei;
+using result_type  = chemist::FragmentedNuclei;
+using frags_type   = chemist::FragmentedNuclei;
+
+namespace { 
+
+auto make_frag_module(const chemist::Nuclei& mol, const frags_type& rv) {
+    return pluginplay::make_lambda<frags_pt>([=](auto&& mol_in) {
+        REQUIRE(mol_in == mol);
+        return rv;
+    });
+}
+} // namespace
+
+TEST_CASE("Dimer Maker") {
+
+    auto mm        = initialize();
+    auto& mod      = mm.at("Dimer Maker");
+
+    SECTION("Empty Nuclei") {
+        input_type nukes;
+        result_type corr(nukes);
+        mod.change_submod("Fragmentation", make_frag_module(nukes, corr));
+        result_type frags = mod.run_as<my_pt>(nukes);
+        REQUIRE(corr.operator==(frags));
```

Comment:
```suggestion
        REQUIRE(corr == frags);
```
You shouldn't have to call the member function explicitly like that.